apps/jobs/utils.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import status
+from rest_framework.response import Response
+from challenges.models import (
+    ChallengePhase,
+    Challenge,
+)
+from participants.models import (ParticipantTeam,)
+
+
+def get_challenge_object(challenge_pk):
+    try:
+        Challenge.objects.get(pk=challenge_pk)
+    except Challenge.DoesNotExist:
+        response_data = {'error': 'Challenge does not exist'}
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+
+def get_challenge_phase_object(challenge_phase_pk):
```

Comment:
Also, I think these utils functions should not be a part of this PR. They should go in a separate PR on per app basis(only raise PR for functions which are required as of now in this PR).